src/services/control-plane/app/schemas/auth.ts

```diff
@@ -0,0 +1,8 @@
+export interface AuthDetails{
+     token:string;
+     refreshToken:string;
+}
```

Comment:
Agregar espacios entre unidades lógicas
```suggestion
}

```